apps/bestofjs-nextjs/src/components/search-palette/search-palette.tsx

```diff
@@ -123,7 +124,8 @@ export function SearchPalette({ allProjects, allTags }: SearchProps) {
   const onSelectProject = (itemValue: string) => {
     const projectSlug = itemValue.slice("project/".length);
     const project = allProjects.find((project) => project.slug === projectSlug);
-    setSelectedItem({ type: "project", value: project! });
+    invariant(project);
```

Comment:
Not including an error message for invariant to throw?

Reply:
It's not required as I use `tiny-invariant` for its "type narrowing" feature, to avoid non null assertions (rather than for run-time checks) but I can add a message, OK, let's be safe :)

https://github.com/alexreardon/tiny-invariant